keyboardClass.js

```diff
@@ -0,0 +1,241 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+class Keyboard {
+    constructor(lang, langAdd, buttonList, container) {
+        this.language = localStorage.getItem('lang') !== null ? localStorage.getItem('lang') : lang;
+        this.languageAlter = this.language === 'en' ? 'ru' : 'en';
+        this.btnList = buttonList;
+        this.domElement = container;
+        this.upCase = false;
+        this.shiftPressed = false;
+        this.capsPressed = false;
+        this.ctrlPressed = false;
+        this.altPressed = false;
```

Comment:
It looks like another control key will lead to another variable. What if there are 100 control keys that we need to track? Let's find another approach